client/client_test.go

```diff
@@ -252,6 +253,46 @@ func TestClient_HasNodeChanged(t *testing.T) {
 	}
 }
 
+func TestClient_Fingerprint_Periodic(t *testing.T) {
+	if _, ok := driver.BuiltinDrivers["mock_driver"]; !ok {
+		t.Skip(`test requires mock_driver; run with "-tags nomad_test"`)
+	}
+	t.Parallel()
+
+	c1 := testClient(t, func(c *config.Config) {
+		c.Options = map[string]string{
+			"test.shutdown_periodic_after":    "true",
```

Comment:
Can you make this a constant on the mock driver.
"test.mock_driver.shutdown_periodic..."